lib/webrick/cgi.rb

```diff
@@ -160,7 +160,7 @@ def service(req, res)
         __send__(method_name, req, res)
       else
         raise HTTPStatus::MethodNotAllowed,
-              "unsupported method `#{req.request_method}'."
+              "unsupported method '#{req.request_method}'."
```

Comment:
I think it's even better (safer) to use `req.request_method.inspect` here and in other places. What do you think?

Reply:
I agree that it is safer.
I think it would be better to do it in a separate pull request.

Reply:
Okay, do you want to open a new pull request?

Reply:
I don't have time to do it, so I'll leave it to you.